src/OpenTelemetry.Exporter.Zipkin/ZipkinExporterHelperExtensions.cs

```diff
@@ -43,8 +43,19 @@ public static TracerProviderBuilder AddZipkinExporter(this TracerProviderBuilder
             configure?.Invoke(exporterOptions);
             var zipkinExporter = new ZipkinExporter(exporterOptions);
 
-            // TODO: Pick Simple vs Batching based on ZipkinExporterOptions
-            return builder.AddProcessor(new BatchExportProcessor<Activity>(zipkinExporter));
+            if (exporterOptions.ExporterType == ZipkinExporterType.SimpleExportProcessor)
```

Comment:
If someone sets the type to simple export processor, and set the batch size to 100, what do we do?

Reply:
We would just do a SimpleExport. The batchSize would be ignored. The idea is to have the documentation updated to inform that when SimpleExport is used the other BatchProcessor related options if provided would be ignore.

@cijothomas  comments?

Reply:
Sounds reasonable.

Reply:
This is similar to https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/sdk-environment-variables.md ENV variables. A setting like `OTEL_EXPORTER_JAEGER_ENDPOINT` is only evaluated **if** OTEL_EXPORTER is jaeger, else ignored.